Tests/iOS/Tests/Extensions/StringCacheTests.swift

```diff
@@ -4,18 +4,28 @@ import XCTest
 final class StringCacheTests: XCTestCase {
   /// Tests that it decodes from NSData
   func testDecode() {
-    let string = self.name
-    let data = string!.data(using: String.Encoding.utf8)!
+    #if os(tvOS)
```

Comment:
I have no idea, but tests under tvOS has `name` as optional 😲

Reply:
This is weird.